docs/data/load-balanced-gridftp.md

```diff
@@ -31,23 +31,27 @@ The number of GridFTP servers that you should run is determined first and foremo
 
 #### Shared file system
 
-The number of GridFTP servers can also be determined by your hardware needs, which are determined by your choice of shared file system. If you choose a POSIX-based shared file system, plan for machines with more cores, or more GridFTP hosts to distribute the CPU load. If you are running [GridFTP with Hadoop](install-hadoop#standalone-gridftp-node-installation), plan for machines with more memory, or more GridFTP hosts to distribute the memory load.
+The number of GridFTP servers can also be determined by your hardware needs and by your choice of shared file system. If you choose a POSIX-based shared file system, plan for machines with more cores, or more GridFTP hosts to distribute the CPU load. If you are running [GridFTP with Hadoop](install-hadoop#standalone-gridftp-node-installation), plan for machines with more memory, or more GridFTP hosts to distribute the memory load.
 
 !!! note
     If you determine that you need only a single GridFTP host, you do not need load balancing. Instead, follow the [standalone-GridFTP installation guide](gridftp).
 
 ### Load balancer(s)
 
-In the recommended direct routing mode, load balancers simply rewrite initial packets from a given request so the hardware requirements are minimal. When choosing load balancer hosts, aim for stability. If your chosen host is unstable or if you do not want to introduce downtime for operating system or hardware updates, one additional load balancer as a backup should be sufficient.
+In the recommended direct routing mode, load balancers simply rewrite initial packets from a given request so the hardware requirements are minimal. When choosing load balancer hosts, aim for stability. If your chosen host is unstable or if you do not want to introduce downtime for operating system or hardware updates, at least one additional load balancer will be needed as a backup.
 
 Preparing the GridFTP Servers
 -----------------------------
 
-Before adding your GridFTP hosts to the load-balanced system, each host requires the GridFTP software, special certificates, and load-balancing configuration.
+Before adding your GridFTP hosts to the load-balanced system, each host requires the following:
 
-### Acquiring service certificate(s)
+* GridFTP software
+* special host certificates
+* load-balancing configuration.
```

Comment:
Could you capitalize each of these bullets and get rid of the period from the last one?